guides/material-3.md

```diff
@@ -78,6 +78,34 @@ used with the `primary` and `tertiary` options:
 - `$m3-violet-palette`
 - `$m3-rose-palette`
 
+For more customization, you can run a schematic to generate a scss file with
+theme(s) that use custom colors. You can specify a color to represent the
+primary color palette and the rest of the color palettes (secondary, tertiary,
+neutral) are generated from Material. The generated color palettes are
+optimized to have enough contrast to be more accessible, for more context you
+can read here: [Science of Color Design](https://material.io/blog/science-of-color-design).
+
+You can also specify colors to represent the other color palettes if you want
+more customization. It is recommended to choose colors that are contrastful,
+Material has more detailed guidance for [accessible design](https://m3.material.io/foundations/accessible-design/patterns).
+
+```shell
+ng generate @angular/material:m3-theme
+```
+
+You can then import the generated theme scss file to use `$m3-light-theme` and/or
+`$m3-dark-theme` to where you apply your themes.
+
+```scss
+@import './path/to/m3-theme';
+
+html {
+  // Apply the base theme at the root, so it will be inherited by the whole app.
+  @include mat.all-component-themes($m3-light-theme);
```

Comment:
This example wouldn't work for users - it directly overrides the light theme with dark theme

Also, we don't recommend users load the `all-component-themes`. It'd be best to show it applied to individual components they use. 

Ideally this example looks like a real theme file, e.g. it's missing calls to `core`

Reply:
Thanks for catching that!